class-work/04-monitor-report.qmd

```diff
@@ -0,0 +1,100 @@
+---
```

Comment:
My first instinct was to make this a Jupyter notebook, but the rendering to report is sooo much nicer for a `qmd` over `ipynb`. The API endpoint iframe renders really nicely in Quarto, which was my main concern!

Reply:
Nice! 🙌